pkg/grizzly/server.go

```diff
@@ -40,7 +41,7 @@ var upgrader = websocket.Upgrader{
 	CheckOrigin:     func(r *http.Request) bool { return true },
 }
 
-func NewGrizzlyServer(registry Registry, parser Parser, parserOpts ParserOptions, resourcePath string, port int, openBrowser, watch, onlySpec bool, outputFormat string) (*Server, error) {
+func NewGrizzlyServer(registry Registry, resourcePath string, port int) (*Server, error) {
```

Comment:
Looking at the changes, this could follow the "functional options pattern",  WDYT?

Reply:
It would, except, I'd need to move the server to its own package, otherwise we'd get `grizzly.withContext()` rather than `server.withContext()`.